assets/stylesheets/components/_avatar-group.scss

```diff
@@ -0,0 +1,22 @@
+$base-size: 16px;
+
+%avatar-dimensions {
+	@for $i from 1 through 6 {
+		&.avatar-size-#{$i} {
+		  img {
+  			width: $base-size * $i;
```

Comment:
Please cleanup the spacing in this file... we've standardized on soft tabs, 2 spaces per tab.